pandas-stubs/core/series.pyi

```diff
@@ -685,7 +685,7 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     @overload
     def apply(
         self,
-        func: Callable[..., Hashable],
+        func: Callable[..., Scalar | Sequence | Mapping],
```

Comment:
Don't we basically assume everywhere in this code base that the expectation is that dataframes and series only store hashable/scalar types, and NOT things like `dicts`, or arbitrary objects for the matter?

Reply:
Yes, but in this case, it doesn't matter, because he is defining a `Callable` that returns types, that `apply` will then use to return a `Series`.